Makefile

```diff
@@ -85,12 +85,13 @@ check-docs: generate-docs ## Check whether reference documentation needs to be g
 
 generate-docs: ## Generate command-line reference documentation
 	rm -rf ./docs/commands/*
-	go run ./cmd/rhoas docs --dir ./docs/commands --file-format adoc
+	go run ./cmd/rhoas docs --dir ./docs/commands --file-format md
```

Comment:
Markdown by default - same rules as with adoc (already replaced in the PR).
Downstream gets adoc files that are pushed to separate branch that no one will see.

Reply:
tip: `md` is already the default value, so you could omit it, if you wanted to.